app/pages/articles/[articleId].tsx

```diff
@@ -135,18 +143,119 @@ const ArticleDetails = (props) => {
+                            ?._avg.response!.toFixed(1)}
+                        </div>
+                        <Rating
+                          readOnly
+                          value={
+                            articleScores.find((score) => score.questionId === category.questionId)?._avg
+                              .response! / ratingScaleMax
+                          }
+                          precision={0.1}
+                          max={1}
+                          sx={{
+                            fontSize: 60,
+                            color: smallStarColor,
+                          }}
+                          emptyIcon={<StarIcon style={{ opacity: .40, color: "#737373" }} fontSize="inherit" />}
+                        />
+                      </div>
+                      <div className="w-16 text-center text-[0.7rem] text-gray-darkest dark:text-white">
+                        {category.questionCategory}
+                      </div>
+                    </div>
+                  ) : (
+                    // When the category does not have a star (N/A)
+                    <div key={category.questionId} className="text-center mx-2">
+                      <div className="flex items-center justify-center">
+                        <div className="absolute text-gray-darkest font-semibold text-xs opacity-70 z-50">N/A</div>
+                        <Rating
+                          readOnly
+                          value={0}
+                          precision={0.1}
+                          max={1}
+                          sx={{
+                            fontSize: 60,
+                            color: smallStarColor,
+                          }}
+                          emptyIcon={<StarIcon style={{ opacity: 0.30, color: "#737373" }} fontSize="inherit" />}
+                        />
+                      </div>
+                      <div className="w-16 text-center text-[0.7rem] opacity-70 text-gray-darkest dark:text-white">
+                        {category.questionCategory}
                       </div>
+                    </div>
+                  )
+                )}
+              </div>}
+          </>
+        }
+        <Accordion
+          expanded={isAccordionOpen}
+          onChange={() => toggleAccordion()}
+          sx={{
+            backgroundColor: "transparent",
+            display: "hidden",
+            border: 0,
+            borderColor: "transparent",
+            boxShadow: 0,
+            width: 'auto',
+            padding: 0,
+            transitionDelay: 0,
+            msTransitionDuration: 0,
+            ":before": {
+              display: "none",
+            }
+          }}
+          onClick={() => toggleAccordion()}
+        >
+          <AccordionSummary
+            expandIcon={undefined}
+            aria-controls="panel1a-content"
+            id="panel1a-header"
+            classes={{ root: "hidden" }}
```

Comment:
This should be removing the space when the accordion is collapsed. It is in the local, but not in the deployed version. 

Local:

<img width="624" alt="image" src="https://user-images.githubusercontent.com/17035406/184739409-98a540a5-a2be-484d-a1db-aff9ab780b37.png">

Deployed:
<img width="695" alt="image" src="https://user-images.githubusercontent.com/17035406/184739782-7cf692b1-056e-4981-80c9-695f8fd669d8.png">


Anyway, I think this is ok to forgo right now